mods/cnc/weapons/other.yaml

```diff
@@ -1,5 +1,5 @@
-Flamethrower:
-	ValidTargets: Ground, Trees
+^FlameWeapon:
+	ValidTargets: Ground, Water, Trees
```

Comment:
In my opinion yes. It doesn't affect gameplay really, but players should be allowed to force-fire on water if they want to.